SonarQube.TeamBuild.PreProcessor/SonarLintAnalyzerProvider.cs

```diff
@@ -104,27 +111,33 @@ private static bool TryCreateRuleset(ISonarQubeServer server, TeamBuildSettings
             return false;
         }
 
-        private static string TryGetProfileExportForProject(ISonarQubeServer server, string projectKey, ILogger logger)
+        private static bool TryGetProfileExportForProject(ISonarQubeServer server, string projectKey, ILogger logger, out string rulesetContent, out string parametersContent)
         {
-            string profileContent = null;
-
             string qualityProfile;
-            if (server.TryGetQualityProfile(projectKey, CSharpLanguage, out qualityProfile))
+            if (!server.TryGetQualityProfile(projectKey, CSharpLanguage, out qualityProfile))
             {
-                if (server.TryGetProfileExport(qualityProfile, CSharpLanguage, SonarLintProfileFormatName, out profileContent))
-                {
-                    logger.LogDebug(Resources.SLAP_ProfileExportFound, projectKey);
-                }
-                else
-                {
-                    logger.LogDebug(Resources.SLAP_ProfileExportNotFound, projectKey);
-                }
+                logger.LogDebug(Resources.SLAP_NoProfileForProject, projectKey);
+                rulesetContent = null;
+                parametersContent = null;
```

Comment:
It might be simpler just to set *rulesetContent* and *parametersContent* to null at the start of the method.